include/functions.h

```diff
@@ -1347,14 +1347,14 @@ void func_800AA178(u32);
 View* View_New(GraphicsContext* gfxCtx);
 void View_Free(View* view);
 void View_Init(View*, GraphicsContext*);
-void func_800AA358(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up);
-void func_800AA3F0(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up);
+void View_LookAt(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up);
```

Comment:
This updates more than the `lookAt` position so the name may be slightly odd at a glance, although I see the parallel between this and `gluLookAt` from OpenGL and `guLookAt(F)` from libultra which do the same thing. I don't necessarily suggest any changes but it may be worth pondering.

Reply:
Yeah, I did deliberate a bit on this, but I think it's okay. The only other things it's doing are some quick input sanitization and setting the dirty flag, which I feel are acceptable things to do in a setter without explicitly calling them out in the name.